src/uwloc/data/main.py

```diff
@@ -0,0 +1,53 @@
+import glob
+import os
+
+import typer
+from typing_extensions import Annotated
+
+from .tile import MAX_HOURS, MAX_UNITS, create, init_tiledb, write_row
+from .wav import read_wav
+
+main = typer.Typer()
```

Comment:
Usually this is called `app` since it signifies a Typer `app` rather than just a normal `main` function, curious what your reasoning behind calling this `main`?

Reply:
I'm sure it just came from a sample :). Maybe because there can be subcommands so the top one got called main? I don't feel strongly either way.

Reply:
Let's change this to `app` since you can have a `def main` that captures an actual main routine. Naming this to app would be clear that this is a Typer app instance.